onecodex/viz/_functional.py

```diff
@@ -54,39 +54,54 @@ def plot_functional_heatmap(
                 metric = FunctionalAnnotationsMetric.Cpm
         metric = FunctionalAnnotationsMetric(metric)
 
-        df = self._to_normalized_functional_df(annotation=annotation, metric=metric)
```

Comment:
The ungrouped / unmapped features are already filtered out in the backend

Reply:
I was not sure about client lib. But it is going to use the same API endpoint. So it is fine to remove it.